src/Header.messages.jsx

```diff
@@ -6,6 +6,11 @@ const messages = defineMessages({
     defaultMessage: 'Courses',
     description: 'Link to the learner course dashboard',
   },
+  'header.links.exams': {
```

Comment:
Its possible to add the translations for other languages, like spanish or Portuguese?